examples/ts/.eslintrc

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
why do we need a custom `eslintrc` here?

Reply:
it enables linting in examples folder too for TS. is not that good?

Reply:
i've copied it from `ts` and made only few edits